NineChronicles.Headless/GraphQLService.cs

```diff
@@ -33,7 +34,7 @@ public class GraphQLService
 
         public const string MagicOnionTargetKey = "magicOnionTarget";
 
-        private GraphQLNodeServiceProperties GraphQlNodeServiceProperties { get; }
+        private static GraphQLNodeServiceProperties? GraphQlNodeServiceProperties { get; set; }
```

Comment:
Since it became a `static` property, `GraphQLService` instances will share unintended `GraphQlNodeServiceProperties`. And it seems there is no part to use `setter`. Shouldn't this part be reverted? 🤔